nginx/README.md

```diff
@@ -104,18 +107,21 @@ sudo nginx -t && sudo nginx -s reload
 
 ### Configuration
 
+
+
 <!-- xxx tabs xxx -->
 <!-- xxx tab "Host" xxx -->
 
 #### Host
 
 To configure this check for an Agent running on a host:
 
-Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized section](#containerized).
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized section][7].
```

Comment:
This is an interesting side effect - looks like the link still works when its rendered this way!